config/routes.rb

```diff
@@ -12,6 +12,7 @@
         get 'find_all', to: 'find#index'
         get 'find', to: 'find#show'
         get 'random', to: 'random#show'
```

Comment:
@lucyconklin since these are more custom routes, it's basically up to you. as far as i can tell, i believe you're following what the project is asking you to do 👍